src/lib.rs

```diff
@@ -2,7 +2,6 @@
 #![deny(
     future_incompatible,
     nonstandard_style,
-    rust_2018_idioms,
```

Comment:
Why did you need to remove this?

Reply:
https://github.com/PyO3/maturin/pull/496#issuecomment-817338064 has some context

Reply:
All right, thanks. It's an issue that only occurs when I use the git version of `PyO3`. It's something I'll need to deal with when `0.14` is released.